docs/reverse_proxy.md

```diff
@@ -80,6 +80,11 @@ server {
         # Increase client_max_body_size to match max_upload_size defined in homeserver.yaml
         client_max_body_size 50M;
     }
+    location /.well-known/matrix/client {
+        return 200 '{"m.homeserver": {"base_url": "https://<matrix.example.com>"}}';
+        default_type application/json;
+        add_header Access-Control-Allow-Origin *;
```

Comment:
do we not need to set the other CORS headers here and for Apache? (https://spec.matrix.org/v1.4/client-server-api/#web-browser-clients)